src/item.cpp

```diff
@@ -1350,6 +1350,11 @@ std::string item::info(bool showtext, std::vector<iteminfo> &dump_ref) const
             dump->push_back(iteminfo("DESCRIPTION",
                 _("This piece of clothing is designed to protect you from harm and withstand a lot of abuse.")));
         }
+        if (is_armor() && has_flag("FRAGILE")) {
+            dump->push_back(iteminfo("DESCRIPTION", "--"));
+            dump->push_back(iteminfo("DESCRIPTION",
+                _("This item is fragile and is more easily damaged.")));
```

Comment:
This line should also mention the fact that it only affects the usage as armor. Something like "this item is fragile and won't protect you for long" or something like that.